kubernetes/samples/scripts/create-weblogic-domain/model-in-image/k8s-domain.yaml.template

```diff
@@ -80,36 +91,39 @@ spec:
     replicas: 2
   # The number of managed servers to start for unlisted clusters
   # replicas: 1
-  #
+  
   # Optional configmap for additional models and variable files
-  #
-  wdtConfigMap : wdt-config-map
-  #
-  # user online update or not (default is false)
-  #
+  wdtConfigMap : domain1-wdt-config-map
+
+  # user model-in-image online update or not (default is false)
   #useOnlineUpdate: "true"
+
   #
   # For life cycle online update, if it is set to true it will cancel the update and error out
   # default is false if not set
   #
   #rollbackIfRequireStart: true
+
   #
-  # wdt domain type for model in image case
-  # Valid WDT_DOMAIN_TYPE_VALUES are 'WLS', 'JRF', and 'RestrictedJRF'
+  # The wdt domain type for model in image case.
+  # Valid values are 'WLS', 'JRF', and 'RestrictedJRF'
   #
   wdtDomainType: "@@DOMTYPE@@"
+
   #
   #
-  # keep jrf schema between updates. If not set default to true
+  # Keep jrf schema between updates. If not set, defaults to true.
   #
   #keepJRFSchema: "true"
```

Comment:
This is no longer used, once the final schema schema is settled, it can be removed

Reply:
ok let's leave it in then for now